README.md

```diff
@@ -137,6 +150,47 @@ Verbs are applied in descending order. You can prefix a transforms or the
 `discard` which prevents inclusion and `cache` (default), which caches the
 file in the build cache. See `frogfs_example.yaml` for example usage.
 
+### Extra Python dependencies
+
+You can ask FrogFS to install additional Python packages into the build-time
+virtualenv used by `mkfrogfs.py`. This helps enable optional features like
+`heatshrink` compression without modifying the project’s default
+`requirements.txt`.
+
+- `python_requirements`: a path or list of paths to `requirements.txt` files.
+- `python_packages`: a list of requirement specifiers (e.g. `pkg`, `pkg==1.2`).
+
+The same options are also supported under a nested `python:` section as
+`python.requirements` and `python.packages`.
```

Comment:
I'm not sure if I like the two different ways of doing things.  In the spirit of the way the YAML is currently implemented, lets go with the toplevel python dict with requirements and packages keys.  Otherwise I really like this.